test/mysql.test.js

```diff
@@ -57,7 +66,9 @@ describe('mysql', function() {
     Post.destroyAll(function() {
       PostWithStringId.destroyAll(function() {
         PostWithUniqueTitle.destroyAll(function() {
-          done();
+          PostWithNumId.destroyAll(function() {
+            done();
```

Comment:
`PostWithNumId.destroyAll(done)` would do the same thing. If you're not doing extra work before the `done` call, might as well save some space on the call stack.